TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Queries/CreateInitialTeacherTrainingQuery.cs

```diff
@@ -0,0 +1,8 @@
+namespace TeachingRecordSystem.Core.Dqt.Queries;
+public record CreateInitialTeacherTrainingQuery : ICrmQuery<Guid>
+{
+    public required Guid? PersonId { get; init; }
+    public required Guid? CountryId { get; init; }
+    public required Guid? ITTQualificationId { get; init; }
+    public required dfeta_ITTResult? Result { get; init; }
```

Comment:
Does this need to be nullable?
```suggestion
    public required dfeta_ITTResult Result { get; init; }
```